_posts/2024/2024-08-14-toolbox.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Toolbox Workshop 2024
+categories:
+  - special
+  - toolbox
+  - veranstaltungen
+
+image: /images/toolbox.png
+author: Christian
+---
+
+In den beiden letzten Wochen der Sommer-Semesterferien veranstaltet PeP et al. wieder den Toolbox-Workshop.
+Die [Anmeldung](https://registration.pep-dortmund.org/events/toolbox23/registration/) ist erforderlich,
+da wir über diesen Weg alle weiteren Informationen verteilen werden.
+
+Der Workshop richtet sich vor allem an die kommenden Drittsemester
+des Physik und Medizinphysik Studiums an der TU Dortmund
+und behandelt die Grundlagen des wissenschaftlichen Arbeitens
+und den Umgang mit den nötigen Werkzeugen.
+
+Vom 23. bis zum 26. September werden Grundlagen der Datenanalyse in der Programmiersprache `Python`
+(NumPy, SciPy, matpltotlib) sowie der Umgang mit der Kommandozeile und der Versionsverwaltung `git` interaktiv vorgestellt.
+
+Ab Freitag, den 27. September, bis zum 02. Oktober stehen die Tage im Zeichen von `LaTeX`.

```

Comment:
vielleicht auch ein Wort dazu, was das ist wie in dem Abschnitt mit python und git